apps/server/src/modules/school/repo/school.repo.integration.spec.ts

```diff
@@ -232,17 +232,50 @@ describe('SchoolMikroOrmRepo', () => {
 				const entity = schoolEntityFactory.buildWithId({ systems, county }, schoolId);
 				await em.persistAndFlush([entity]);
 				em.clear();
-				const schoolDo = SchoolEntityMapper.mapToDo(entity);
+				const schoolDos = SchoolEntityMapper.mapToDo(entity);
 
-				return { schoolDo, schoolId };
+				return { schoolDos, schoolId };
 			};
 
 			it('should return school', async () => {
-				const { schoolDo, schoolId } = await setup();
+				const { schoolDos, schoolId } = await setup();
 
 				const result = await repo.getSchoolById(schoolId);
 
-				expect(result).toEqual(schoolDo);
+				expect(result).toEqual(schoolDos);
+			});
+		});
+	});
+
+	describe('getSchoolsByIds', () => {
+		describe('when entity is not found', () => {
+			it('should throw NotFound', async () => {
```

Comment:
we want to catch that "NotFound" here in the test as well? Otherwise I would reframe it as `should not return any school`

Reply:
Fixed test description.